Formula/qt5.rb

```diff
@@ -39,6 +39,19 @@ class Qt5 < Formula
     sha256 "48ff18be2f4050de7288bddbae7f47e949512ac4bcd126c2f504be2ac701158b"
   end
 
+  # Add missing install targets to Mac QtBase widget examples.  This fixes an
+  # issue with building from source with the `--with-examples` option set which
+  # was introduced between the releases of Qt versions 5.6.0 and 5.6.1 by
+  # <http://code.qt.io/cgit/qt/qtbase.git/commit/?id=5b5b7f97b833e9f91c41285718dc7b566cc37247>.
+  # (Remove this as soon as it is feasible and possible to do so!)
+  #
+  # Core formulae known to fail without this patch (as of 2016-06-19):
+  #   (None known.)
```

Comment:
You can simplify this comment a bit. The other currently used patch is a bit of a special case and thus not the best template. Try to model your patch comment after [this one](https://github.com/Homebrew/homebrew-core/blob/25154bf004f0e2dbc4b9d38dd594605398eacb78/Formula/qt5.rb#L32-L34) that was used in an older version of this formula. (As for when this patch can be expected to be removed, both 5.6.2 and 5.7.1 will most likely contain it.)